tests/test_load.py

```diff
@@ -44,6 +46,8 @@ def dbs(tmpdir_factory, persistent_repository, storage_format):
         dictionary containing root folder for each version
 
     """
+    random.seed(1)
```

Comment:
I do not understand what is the purpose of this statement.

Reply:
I added the following comment:

```python
# Fix seed for audformat.testing
random.seed(1)
```

We need this as we use `audformat.testing.add_table()` and `audformat.testing.add_misc_table()` which add tables with random values. A better approach would be to not use those functions, but define the tables manually, but I was too lazy, to rewrite the whole test just for adding a new argument.